src/garage/tf/algos/npo.py

```diff
@@ -105,13 +107,19 @@ def __init__(self,
                  stop_entropy_gradient=False,
                  entropy_method='no_entropy',
                  flatten_input=True,
+                 fit_baseline_order='after',
                  name='NPO'):
         self._name = name
         self._name_scope = tf.name_scope(self._name)
         self._use_softplus_entropy = use_softplus_entropy
         self._use_neg_logli_entropy = use_neg_logli_entropy
         self._stop_entropy_gradient = stop_entropy_gradient
         self._pg_loss = pg_loss
+        self._fit_baseline_order = fit_baseline_order
+
+        if fit_baseline_order not in ['before', 'after']:
+            raise ValueError('Invalid fit_baseline_order')
```

Comment:
please provide more information to the user. What value did she specify, and what are the valid choices?